### PR TITLE
Add support for 2019.*

### LINKED ISSUE #71: robot-plugin version compatibility with IntelliJ 2019.1
When installing this plugin in an IntelliJ 2019.1 IDE version 0.13, from 2015, is used.  
  
image
  
image
  
I've tried installing version 0.17 and 0.14, both resulted in this error:  
  
image

### PR DESCRIPTION
**Pull request by DjLegolas**
Closes #71.

### CODE REVIEW

**Comment by DjLegolas** [comment]
Test fails because of a fix to `com.intellij.testFramework.ParsingTestCase` at JetBrains/intellij-community@2cb54486e00ea5f09e42ed01403bb4e8de28b892.

**Comment by AmailP** [comment]
Hi @DjLegolas,

thank you very much for your contribution.
Are you able to work on fixing the test case?

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: scala
 
 scala:
-   - 2.11.7
+   - 2.11.12
```
Comment (AmailP) [inline .travis.yml]: If this library update are not necessary for the support of 2019.* I would rather move to another merge request. The same for the update of the gradle plugin and scalatest.

**Comment by DjLegolas** [comment]
> Are you able to work on fixing the test case?

Hmm, I can try, so please explain the purpose of that test.

File: .travis.yml (hunk 1 above)
Comment (DjLegolas) [reply .travis.yml]: Done.

**Comment by AmailP** [comment]
Hehehe... that is indeed a good question. I think is just some dead code that entered the codebase by mistake. I will just delete it :-)

**Comment by AmailP** [comment]
Thank you very much for your contribution.

I have just merged your changes to master and released a new version of the plugin:
https://github.com/AmailP/robot-plugin/pull/73

**Pull request closed without merging**